custom_model_runner/datarobot_drum/drum/gpu_predictors/triton_predictor.py

```diff
@@ -22,7 +22,7 @@
 from datarobot_drum.drum.language_predictors.base_language_predictor import BaseLanguagePredictor
 from datarobot_drum.drum.utils.drum_utils import DrumUtils
 from google.protobuf import text_format
-from triton.model_config_pb2 import ModelConfig
+from tritonclient.grpc.model_config_pb2 import ModelConfig
```

Comment:
The `tritonclient` already includes the ModelConfig protobuf scheme, so we don't need to maintain it by ourselves.